summoner-cli/examples/cabal-full/.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: ghc ${{ matrix.ghc }}
+    runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        ghc: ["8.4.4", "8.6.5", "8.8.2"]
```

Comment:
Maybe we should put each GHC version on a separate line here as well? 
It will make it consistent with the other CI settings and also allows to change GHC versions easier.